app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -22,16 +15,12 @@ def new
         #{@app.validation_reason} Please contact support if you see this message
       EOT
     end
-  rescue BatchConnect::App::AppNotFound => e
```

Comment:
Hmm... I guess normal exception messages don't add periods, so it would need to be something ugly like:

```ruby
rescue AppNotFound => e
  @validation_reason = "#{e.message}."
```

Or we just remove "Please contact support if you see this message". We actually don't display that anywhere else an error occurs in our apps.